tools/generate_features.py

```diff
@@ -1008,7 +1029,7 @@ def generate_features(
 
         print(f'Computing Fourier stats for {len(period_dict)} algorithms...')
         for algorithm in period_algorithms:
-            if algorithm != 'ELS_ECE_EAOV':
+            if (algorithm != 'ELS_ECE_EAOV') & (algorithm != 'LS_CE_AOV'):
```

Comment:
if algorithm not in ["ELS_ECE_EAOV", "LS_CE_AOV"]:

Reply:
This is better syntax, thanks!